apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -630,6 +630,74 @@ cm_publish_send(Peer, Solution) ->
 		body => ar_serialize:jsonify({Json})
 	})).
 
+%% @doc Fetch the jobs from the pool proxy or coordinated mining exit peer.
+get_jobs(PeerOrURL, PrevOutput, Pool) ->
+	{Peer, Headers, BasePath} =
+		case Pool of
```

Comment:
Can this block and [this one](https://github.com/ArweaveTeam/arweave/pull/515/files#diff-e6210d30a0b0d54dd30e8bfa45bba38a93369c1b0b19419a8f5891e163e45c40R656-R662) be deduplicated?